apps/web/src/pages/Teams/UpdateTeamPage/components/UpdateTeamForm.tsx

```diff
@@ -0,0 +1,166 @@
+import useRoutes from "~/hooks/useRoutes";
+
+import CreateTeamFormFields from "../../CreateTeamPage/components/CreateTeamFormFields";
+import DescriptionWordCounter from "./DescriptionWordCounter";
+
+const TEXT_AREA_ROWS = 4;
+
+type FormValues = {
+  name: string;
+  displayName?: Maybe<LocalizedStringInput>;
+  description?: Maybe<LocalizedStringInput>;
+  contactEmail?: Maybe<Scalars["Email"]>;
+  departments?: Array<Scalars["UUID"]>;
+};
+
+const dataToFormValues = (data: Team): FormValues => {
+  const { departments, displayName, description, ...rest } = data;
+  return {
+    ...omit(rest, ["id", "__typename"]),
+    displayName: omit(displayName, "__typename"),
+    description: omit(description, "__typename"),
+    departments: unpackIds(departments),
+  };
+};
+
+const formValuesToSubmitData = (data: FormValues): UpdateTeamInput => {
+  const { departments, ...rest } = data;
+  return {
+    ...rest,
+    departments: { sync: departments },
+  };
+};
+
+interface UpdateTeamFormProps {
+  team: Team;
+  departments?: Maybe<Array<Maybe<Omit<Department, "teams">>>>;
+  onSubmit: (
+    teamId: Scalars["UUID"],
+    data: UpdateTeamInput,
+  ) => Promise<UpdateTeamMutation["updateTeam"]>;
+}
+
+const UpdateTeamForm = ({
+  team,
+  departments,
+  onSubmit,
+}: UpdateTeamFormProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const navigate = useNavigate();
+
+  const handleSubmit: SubmitHandler<FormValues> = async (data) => {
+    return onSubmit(team.id, formValuesToSubmitData(data))
+      .then(() => {
+        navigate(paths.teamTable());
+        toast.success(
+          intl.formatMessage({
+            defaultMessage: "Team updated successfully!",
+            id: "2ni0ia",
+            description: "Message displayed after a team is updated",
+          }),
+        );
+      })
+      .catch(() => {
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "Error: updating team failed",
+            id: "h486wf",
+            description: "Messaged displayed after updating team fails",
+          }),
+        );
+      });
+  };
+
+  return (
+    <BasicForm
+      onSubmit={handleSubmit}
+      options={{
+        defaultValues: dataToFormValues(team),
+      }}
+    >
+      <div data-h2-flex-grid="base(center, x1, 0)">
+        <CreateTeamFormFields departments={departments} />
+        <div data-h2-flex-item="base(1/2)">
+          <TextArea
+            id="description_en"
+            name="description.en"
+            rows={TEXT_AREA_ROWS}
+            label={intl.formatMessage({
+              defaultMessage: "Organization's short description (English)",
+              id: "TCQhlo",
+              description: "Label for the English team description input",
+            })}
+            rules={{
+              required: intl.formatMessage(errorMessages.required),
+            }}
+          />
+          <DescriptionWordCounter name="description.fr" />
```

Comment:
I think the problem is here. It should be "description.en"

Reply:
Looks like that was already addressed with [this commit](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5743/commits/c929e251da7d646c55f8e962b233f2de85aa8f7f) which has now been merged in.